gcp_variant_transforms/libs/processed_variant.py

```diff
@@ -436,12 +435,16 @@ def add_annotation_data(self, proc_var, annotation_field_name, data):
             self._add_ambiguous_fields(annotations_list, ambiguous)
           alt._info[annotation_field_name] = annotations_list
 
-  def _find_common_alt_ref_prefix(self, proc_var):
+  def _find_common_alt_ref_prefix_char(self, proc_var):
```

Comment:
What is the reasoning behind returning just a single char instead of the prefix? Please add a comment.

Reply:
Please check the updated comment in `_alt_matches_annotation_alt` (line 578 below). That is the right place to explain this as the function doc has examples with links to VEP documentation that describes the ALT mapping logic in details.

So the gist of it is that VCF spec says there should be only one extra base included before an indel variant region. Of course this is not followed in all VCFs and that is what triggered the issue in my new integration tests. In the new platinum file, there are examples that multiple extra bases are added to the variant region and VEP only drops the first base not the whole common part (in the non-minimal mode).

Reply:
I see. Thanks for the explanation. sg.